### DISCUSSION TITLE
Rewind X Seconds

### DISCUSSION THREAD

**Discussion by chookity-pokk** [alphapapa/listen.el#4] [Q&A] [answered]
Thank you for the package! I have used it a bit today and was hoping to find a way to get it to rewind a handful of seconds. I noticed that VLC has some command (below) for jumping forward and backwards some amount of time. I tried using the `listen-shell-command` function on a file I was listening to, to get it to jump back but that doesn't work because it says it doesn't know the command `--global-key-jump-short`. So I tried it with the command being `vlc --global-key-jump-short "Left"` but that just starts a new vlc instance. 

I was hoping for maybe a bit of guidance on what code would need to be changed to allow this to happen then I am happy to open a PR for it, if you'd want that. My elisp knowledge is limited though so I had a bit of a hard time finding a starting point in the files.

```
      --global-key-jump-extrashort <string> 
                                 Very short backwards jump
      --key-jump-extrashort <string> 
                                 Very short backwards jump
      --global-key-jump+extrashort <string> 
                                 Very short forward jump
      --key-jump+extrashort <string> 
                                 Very short forward jump
      --global-key-jump-short <string> 
                                 Short backwards jump
      --key-jump-short <string>  Short backwards jump
      --global-key-jump+short <string> 
                                 Short forward jump
      --key-jump+short <string>  Short forward jump
      --global-key-jump-medium <string> 
                                 Medium backwards jump
      --key-jump-medium <string> Medium backwards jump
      --global-key-jump+medium <string> 
                                 Medium forward jump
      --key-jump+medium <string> Medium forward jump
      --global-key-jump-long <string> 
                                 Long backwards jump
      --key-jump-long <string>   Long backwards jump
      --global-key-jump+long <string> 
                                 Long forward jump
      --key-jump+long <string>   Long forward jump

   Jump sizes:
      --extrashort-jump-size <integer> 
                                 Very short jump length
      --short-jump-size <integer> 
                                 Short jump length
      --medium-jump-size <integer> 
                                 Medium jump length
```

**Comment by alphapapa** [comment, marked as answer]
Hi, please see `C-h f listen-seek RET`.

**Reply by chookity-pokk** [reply]
Sorry for the late response! Thank you. I saw this but the description I saw was that it would just seek a timestamp, I missed the +/- options.
